WebCryptoAPI/encrypt_decrypt/aes.js

```diff
@@ -267,9 +267,11 @@ function run_test() {
         all_promises.push(promise);
     });
 
-    Promise.all(all_promises)
-    .then(function() {done();})
-    .catch(function() {done();})
+    promise_test(function() {
+        return Promise.all(all_promises)
+            .then(function() {done();})
+            .catch(function() {done();})
+    }, "setup");
```

Comment:
.any.worker.js tests automatically call `done()`. This is used to prevent the test suite from preemptively exiting before these promises have resolved.